__tests__/api-categories.test.js

```diff
@@ -1,20 +1,52 @@
 process.env.NODE_ENV = "test";
 const request = require("supertest");
 const  app  = require("../app");
-const db = require("../db/data/test-data");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data")
+
+
+beforeEach(() => {
+	return seed(testData);
+});
 
 afterAll(() => {
 	if (db.end) db.end();
 });
 
+//do beforeaach on onbe line 
+
+
 describe("1. GET /api/categories", () => {
-      test("GET /api/categories returns an object", () => {
+      test("200: GET /api/categories returns an array", () => {
 				return request(app)
```

Comment:
Wooooah your spacing is huge! Not an issue with your code, but maybe have a look at the settings for whatever formatter you're using to make things look a bit nicer.